website/source/docs/platform/k8s/helm.html.md

```diff
@@ -72,6 +72,8 @@ and consider if they're appropriate for your deployment.
 
   * `image` (`string: "vault:latest"`) - The name of the Docker image (including any tag) for the containers running Vault. **This should be pinned to a specific version when running in production.** Otherwise, other changes to the chart may inadvertently upgrade your Vault version.
 
+  * `tlsDisable` (`boolean: true`) - When set to `true`, changes URLs from `https` to `http` (such was the `VAULT_ADDR` environment variable on the Vault pods).
```

Comment:
> such was the `VAULT_ADDR` environment variable on the Vault pods

Such was it what? It's not clear what `tlsDisable` is doing related to `VAULT_ADDR`

Reply:
Good catch, clarified!